src/main/kotlin/fi/metatavu/timebank/api/controllers/UsersController.kt

```diff
@@ -0,0 +1,36 @@
+package fi.metatavu.timebank.api.controllers
+
+import fi.metatavu.timebank.api.impl.translate.UserTranslator
+import fi.metatavu.timebank.api.keycloak.KeycloakController
+import fi.metatavu.timebank.model.User
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+@ApplicationScoped
+class UsersController {
+
+    @Inject
+    lateinit var keycloakController: KeycloakController
+
+    @Inject
+    lateinit var userTranslator: UserTranslator
+
+    fun findUser(userId: UUID): User? {
+        val user = keycloakController.findUserById(userId)
+        if (user != null) {
+            return userTranslator.translate(user)
+        }
+
+        return null
+    }
+
+    fun listUsers(): List<User>? {
+        val users = keycloakController.getUsersResource()
+        if (users != null) {
+            return userTranslator.translate(keycloakController.getUsersResource()!!)
+        }
+
+        return null
+    }
```

Comment:
Return a list of `UserRepresentation`s from `KeycloakController::GetUsersResource` and then pass that through this method e.g.

```kotlin
    fun listUsers(): List<User> {
        return keycloakController.listUsers() 
    }
```

At the moment it seems like this class doesn't serve that big of a purpose but since we know it'll start to handle integrations in the next coming PR's I'd probably write this as of now as well. :)